.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint and tests
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Project checkout
+      uses: actions/checkout@v1
+    - name: Node.js Setup 
+      uses: actions/setup-node@v1.1.0
+    - name: Installation of dependencies
+      run: npm ci
+    - name: Lint
+      run: npm run lint
+    - name: Unit tests
+      run: npm run test:unit
+    - name: E2E tests
+      run: npm run test:e2e
```

Comment:
Should we have an empty line in the end of this file to avoid `No newline at the end of file`?